src/netspresso_trainer/postprocessors/register.py

```diff
@@ -0,0 +1,11 @@
+from typing import Dict, Type
+
+from .classification import ClassificationPostprocessor
+from .detection import YOLOXPostprocessor
+from .segmentation import SegmentationPostprocessor
+
+POSTPROCESSOR_DICT = {
```

Comment:
@illian01 

I think it would be better if we just name the postprocessor with the corresponding task. Will it be different a lot by task heads?

Reply:
Actually, I initialized `DetectionPostprocessor` and renamed to `YOLOXPostprocessor` in [b207dbd](https://github.com/Nota-NetsPresso/netspresso-trainer/pull/223/commits/b207dbdfb64edb73c596cf55f7ca48c14557e52d).

I think it will be different depending on head because they need different functions to produce predictions.
For example, anchor-based models predict delta values for anchors, and anchor-free models predict `cxcywh` directly. This makes us to use two different box decode functions.
Also, differently with CNN based detection models, [detr postprocessor](https://github.com/facebookresearch/detr/blob/3af9fa878e73b6894ce3596450a8d9b89d918ca9/models/detr.py#L258-L286) has no non-maximum suppresion.

If we use detection postprocessor as`DetectionPostprocessor`, we need to assemble proper function depending on detection head. This also can be good choice because most detection postprocess has box decode function or non-maximum suppresion or both.
@deepkyu If you prefer to use `DetectionPostprocessor`, I will fix to that way. :+1:

Reply:
Yeah, I didn' think a lot about detection heads.
The postprocessor could be different depending on the type of detection heads.

However, I still believe that those are distinguished inside the `DetectionPostprocessor`, cause there's possibility that the same postprocessor code could be used with various heads.

For now, let's proceed and discuss in later :) Thank you for consideration.